skins/cat17/templates/Display_Page_Layout.twig

```diff
@@ -1,6 +1,6 @@
 {% extends 'Base_Layout.html.twig' %}
 
-{% block page_identifier %}{$ page_id $}{% endblock %}
+{% block page_identifier %}page-{$ page_id $}{% endblock %}
```

Comment:
I don't particularly like the prefix but without it in the CSS selectors I am afraid to involuntarily match styles. Will follow-up and consolidate once all pages work the same way (prepend it to only the identifier rendered onto the body, not the page_identifier variable).